core/commands/charmil.go

```diff
@@ -10,13 +10,20 @@ func AttachCharmilCommands(hostRoot *cobra.Command) error {
 		return nil
 	}
 
-         // Placeholder command. To be replaced later by actual commands
+	// Placeholder command. To be replaced later by actual commands
 	cmd := &cobra.Command{
 		Use:     "charmil",
 		Short:   "built in charmil commands",
 		SilenceUsage: false,
 	}
 
+	// charmil custom commands
```

Comment:
Confusing - Charmil commands will be dedicated to end user.
Like for registry it will be view plugins to install or manage config.
Validator is not charmill SDK - it is separate component